mqemitter-redis.js

```diff
@@ -54,15 +63,15 @@ function MQEmitterRedis (opts) {
   })
 
   this.subConn.on('error', function (err) {
-    that.state.emit('error', err)
+    that.state.emit('connectionError', err)
```

Comment:
Why this became `connectionError`? The use of `error` was intentional to crash the mqemitter.

Reply:
Fixed